clusterloader2/testing/load/config.yaml

```diff
@@ -74,7 +74,11 @@
 {{$EXEC_TIMEOUT := DefaultParam .CL2_EXEC_TIMEOUT "3600s"}}
 {{$SLEEP_AFTER_EXEC_DURATION := DefaultParam .CL2_SLEEP_AFTER_EXEC_DURATION "0s"}}
 
-{{$latencyPodImage := DefaultParam .CL2_LATENCY_POD_IMAGE "registry.k8s.io/pause:3.9"}}
+## Image Repositories
+{{$registryK8sRepo := DefaultParam .CL2_REGISTRY_K8S_REPO "registry.k8s.io"}}
+{{$gcrRepo := DefaultParam .CL2_GCR_REPO "gcr.io"}}
```

Comment:
This field and above seem different registries for different images, I'll defer to this repository's maintainers here, but maybe we should try to find some naming that clarifies a bit more where they're set?